lib/nrf_modem_lib/netif/netif.c

```diff
@@ -452,3 +450,14 @@ int lte_connectivity_options_get(struct conn_mgr_conn_binding *const if_conn, in
 
 	return 0;
 }
+
+/* Bind L2 connectity APIs.
+ * extern in nrf91_sockets.c
+ */
+struct conn_mgr_conn_api conn_api = {
+	.init = nrf_modem_lib_netif_init,
+	.connect = nrf_modem_lib_netif_connect,
+	.disconnect = nrf_modem_lib_netif_disconnect,
+	.set_opt = nrf_modem_lib_netif_options_set,
+	.get_opt = nrf_modem_lib_netif_options_get,
+};
```

Comment:
Shouldn't the actual Zephyr `net_if` definition for nRF91 also be moved here, if we want to abstract Zephyr integration stuff? Currently done in https://github.com/nrfconnect/sdk-nrf/blob/main/lib/nrf_modem_lib/nrf91_sockets.c#L1129

Reply:
It could be done but we'd need to extern all the nrf91 socket APIs, right?